### DISCUSSION TITLE
Draft pull requests for private repos in user account

### DISCUSSION THREAD

**Discussion by fawaf** [community/community#81517] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

will draft pull requests be supported for private repos in user accounts at some point?

**Comment by SMGolamAlif** [comment]
Hey, so you're wondering if someday you'll be able to use draft pull requests for your private repos in your user account. Well, fingers crossed! Right now, it might not be a thing, but who knows what the future holds? GitHub and similar platforms are always tweaking stuff and rolling out new features. Keep an eye out for updates – your wish might just come true! 🤞✨

**Reply by objecttothis** [reply]
At least get rid of the link teasing draft PR if you aren't going to make it available to private repos. It confuses users into searching the page for how to turn the PR into a draft when it's not possible because it's a private repo.

![image](https://github.com/community/community/assets/17935339/486ef3db-235b-4092-b85d-157538659c0d)

**Comment by faridnec** [comment]
draft pull requests are not officially supported for private repositories in user accounts. This functionality is currently limited to organization-owned repositories and public repositories. but for a potential solution, you can use your local Git environment and a text editor to create and refine pull request content before officially opening it -> push your changes to a separate branch in your private repository -> When ready, create the pull request from that branch.

hope this helps

**Comment by cr333** [comment]
Quick update from the GitHub Blog (1 May 2025):
[Draft pull requests are now available in all repositories](https://github.blog/changelog/2025-05-01-draft-pull-requests-are-now-available-in-all-repositories/)
